ui/tests/integration/components/replication-actions-test.js

```diff
@@ -111,16 +121,32 @@ module('Integration | Component | replication actions', function(hooks) {
       });
       this.set('storeService.capabilitiesReturnVal', ['root']);
       await render(
-        hbs`{{replication-actions model=model replicationMode=replicationMode selectedAction=selectedAction onSubmit=(action onSubmit)}}`
+        hbs`
+        <div id="modal-wormhole"></div>
+        {{replication-actions model=model replicationMode=replicationMode selectedAction=selectedAction onSubmit=(action onSubmit)}}
+        `
+      );
+      assert.equal(
+        find('h4').textContent.trim(),
+        headerText,
+        `${testKey}: renders the correct component header (${oldVersion})`
       );
-
-      assert.equal(find('h4').textContent.trim(), headerText, `${testKey}: renders the correct component`);
 
       if (typeof fillInFn === 'function') {
         await fillInFn.call(this);
       }
-      await click('[data-test-confirm-action-trigger]');
-      await click('[data-test-confirm-button]');
+      if (oldVersion) {
```

Comment:
nice thinking on these tests! hopefully we can delete the old versions very soon ಠ‿ಠ